src/NetAppFiles/NetAppFiles/help/New-AzNetAppFilesVolume.md

```diff
@@ -175,6 +178,21 @@ Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
+### -ProtocolType
+A hashtable array which represents the export policy
+
+```yaml
+Type: List[String]
```

Comment:
@leonardbf was this regenerated using the `platyPS` module? This type should automatically be resolved to `System.String[]` using `platyPS`.

Reply:
Ah, no not after the previous review comment fix. Updated.